### ISSUE TITLE
Grouping failing when data contains a property called 'group'

### ISSUE THREAD

**Issue by adecler** [FormidableLabs/victory#1303]
# Bugs and Questions

### Checklist

- [x] This is not a `victory-native` specific issue. (Issues that only appear in `victory-native` should be opened [here](https://github.com/FormidableLabs/victory-native/issues/new))

- [x] I have read through the [FAQ](https://formidable.com/open-source/victory/docs/faq) and [Guides](https://formidable.com/open-source/victory/guides/) before asking a question

- [x] I am using the latest version of Victory

- [x] I've searched open issues to make sure I'm not opening a duplicate issue

### The Problem

If the data contains a property called 'group', only the first item in each group appears in the chart.

The problem comes from `getDataFromChildren` method in the `wrapper.js` file:

![image](https://user-images.githubusercontent.com/16853390/57140181-0c06fe00-6daf-11e9-8f06-bbb6391c8a42.png)

the call to `uniqBy()` is then using the value from the group property instead of the group index created earlier. Can we use '_group' instead on line 209 and 215 ?

### Reproduction

I did a quick tweak of the first example in the VictoryGroup documentation page to add a group variable. Here's the result:

```
<VictoryChart>
  <VictoryGroup offset={20}
    colorScale={"qualitative"}
  >
    <VictoryBar
      data={[{ x: 1, y: 1, group:'A' }, { x: 2, y: 2, group:'A' }, { x: 3, y: 5, group:'A' }]}
    />
    <VictoryBar
      data={[{ x: 1, y: 2, group:'B' }, { x: 2, y: 1, group:'B' }, { x: 3, y: 7, group:'B' }]}
    />
    <VictoryBar
      data={[{ x: 1, y: 3, group:'C' }, { x: 2, y: 4, group:'C' }, { x: 3, y: 9, group:'C' }]}
    />
  </VictoryGroup>
</VictoryChart>
```

**Comment by boygirl** [comment]
@adecler yes, I will change the property to `_group`. Thanks for making my job easy with this thorough issue!

**Comment by boygirl** [comment]
released in `victory@32.2.2`

**Comment by adecler** [comment]
@boygirl , thanks for the fast response 👍
